### DISCUSSION TITLE
Played/unplayed filters in song selection

### DISCUSSION THREAD

**Discussion by brandondong** [ppy/osu#25809] [Ideas] [answered]
In stable's main song selection menu, you can filter beatmaps by played date (e.g. `played<30`, `unplayed=`, etc.). Please correct me if I'm wrong but this doesn't seem to work on lazer? Other search filters work like `stars>4` and `ar=9`.

Was this particular filter not implemented? Are there plans to add it?

**Comment by bdach** [comment, marked as answer]
pending implementation, stuck in review (https://github.com/ppy/osu/pull/23129)

**Reply by brandondong** [reply]
I have a vested interest in seeing this feature make it to lazer. Since the original PR author appears to be inactive, I have extended their branch in a new PR (https://github.com/ppy/osu/pull/25885) with the most up-to-date master changes merged and any outstanding comments hopefully now addressed.

If there is any more feedback, I can follow through with the necessary changes in order to see this PR to completion.
